src/components/StaffGrid/index.jsx

```diff
@@ -41,7 +41,7 @@ export default class StaffGrid extends Component {
           {provided => (
             <div ref={provided.innerRef} className="staff-grid row" {...provided.droppableProps}>
               {this.state.staffList.map((staffer, index) => (
-                <Draggable draggableId={staffer.uuid} index={index}>
+                <Draggable draggableId={staffer.given_name} index={index}>
```

Comment:
I don't think it's safe for us to assume that the `given_name` will be unique

Reply:
That's a good call. The screenreader uses the id for reading out the item it's selected, so I'm not sure `uuid` works either. I guess that's something to worry about later.

Reply:
Maybe use `getStafferName` for the id? Seems unlikely that they'll have the same given and family name